README.md

```diff
@@ -14,7 +14,7 @@ work around that and make the second (non-header) row also bold. -->
 | **Rust**      | [![Rust coverage][coveralls-badge]][coveralls-link] |
 
 <!-- markdownlint-disable line-length -->
-[buildkite-badge]: https://badge.buildkite.com/15a8faccee0d5b5ab1af7e75eb6f9daf2d493c543fbc67dce5.svg?branch=master
+[buildkite-badge]: https://badge.buildkite.com/16896a68bd8fba45d7b41fd608f26f87c726da10f7f24694a0.svg?branch=master
 [buildkite-link]: https://buildkite.com/oasisprotocol/oasis-core-ci
 [github-ci-lint-badge]: https://github.com/oasisprotocol/oasis-core/workflows/ci-lint/badge.svg
 [github-ci-lint-link]: https://github.com/oasisprotocol/oasis-core/actions?query=workflow:ci-lint
```

Comment:
I just noticed it would be useful to change the link of the GitHub Actions' ci-lint workflow badge to https://github.com/oasisprotocol/oasis-core/actions?query=workflow:ci-lint+branch:master to limit the results to the `master` branch, otherwise it is confusing if the badge says the workflow is passing and the link points to failed workflow runs of PRs...

Reply:
Good idea, noticed that a while back but then forgot about it.